SimPPS/Configuration/python/SimPPS_cff.py

```diff
@@ -0,0 +1,24 @@
+import FWCore.ParameterSet.Config as cms
+
+# PPS Digitization
+from SimPPS.PPSPixelDigiProducer.RPixDetDigitizer_cfi import *
+from SimPPS.RPDigiProducer.RPSiDetDigitizer_cfi import *
+RPixDetDigitizerTask=cms.Task(RPixDetDigitizer)
+RPSiDetDigitizerTask=cms.Task(RPSiDetDigitizer)
+
+ctppsDigiTask = cms.Task()
+ctppsDigi = cms.Sequence(ctppsDigiTask)
```

Comment:
Is the `ctppsDigi` Sequence used anywhere? If not, it could be removed (after which `ctppsDigiTask` could be renamed to `ctppsDigi` if desired).

Reply:
Done, but for uniformity reason, the name ctppsDigiTask was kept.